src/app/main.cpp

```diff
@@ -135,6 +135,8 @@ int main(int argc, char *argv[])
 
     try {
         // Create Application
+        QApplication::setAttribute(Qt::AA_UseHighDpiPixmaps);
+        QApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
```

Comment:
Move the changes to here (below this line actually):
https://github.com/qbittorrent/qBittorrent/blob/316f34cbf5e956cd1381d85aff30f9eafd4c4714/src/app/application.cpp#L142
And enclose it with `#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))`.

Reply:
Also it is better to let users have the ability to override:
```c++
#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
    if (qgetenv("QT_AUTO_SCREEN​_SCALE_FACTOR").isEmtpy())
        setAttribute(Qt::AA_EnableHighDpiScaling, true);
#endif
```

Reply:
agreed, it is better to keep an option to disable this feature, at least for compatibility reasons, but don't agree with Qt version check condition, see comment below https://github.com/qbittorrent/qBittorrent/pull/12102#issuecomment-594114889

Reply:
it is impossible to put `setAttribute(Qt::AA_EnableHighDpiScaling, true);` where you suggested... it is too late...
```
Attribute Qt::AA_EnableHighDpiScaling must be set before QCoreApplication is created.
```